### PR TITLE
SWL-3216 IPV6 address pretty_print function.

### PR DESCRIPTION
**Pull request by poolakiran**
Reviewer: @kenchiang

### CODE REVIEW

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_docker/69/

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/pr-520-jenkins-loxigen_pull_req_artifacts-63

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_artifacts/63/

File: py_gen/templates/util.py [hunk 1]
```diff
@@ -52,6 +52,9 @@ def pretty_mac(mac):
 def pretty_ipv4(v):
     return "%d.%d.%d.%d" % ((v >> 24) & 0xFF, (v >> 16) & 0xFF, (v >> 8) & 0xFF, v & 0xFF)
 
+def pretty_ipv6(v):
+    return ":".join(["%0.2x%0.2x" % (ord(v[i]), ord(v[i+1])) for i in range(0, len(v), 2)])
```
Comment (kenchiang) [inline py_gen/templates/util.py]: How about socket.inet_ntop ?
https://docs.python.org/2/library/socket.html

Comment (poolakiran) [inline py_gen/templates/util.py]: I am not sure if we are allowed to use library functions.

Comment (kenchiang) [inline py_gen/templates/util.py]: ok; lgtm

**Comment by poolakiran** [comment]
SELF-ACK

**Comment by poolakiran** [comment]
ACK

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  Log file at http://vss1/abat/2016.09.20.2057-m.loxigen.master/abat.log

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
Log file at http://vss1/abat/2016.09.20.2057-m.loxigen.master/abat.log
/cc

**Pull request merged**